worldengine/basic_map_operations.py

```diff
@@ -1,9 +1,12 @@
 import math
-import random
+import numpy
 
 
-def random_point(width, height):
-    return random.randrange(0, width), random.randrange(0, height)
+def random_point(width, height, randint_substitute=None):
```

Comment:
Neat, but we don't use the extra feature in our code.  Do you see others using this function? I'm not sure as to the utility of this and others can just as easily write their own in their code.